jekyll/_cci2/local-jobs.md

```diff
@@ -77,6 +77,10 @@ The CLI tool automatically checks for updates and will prompt you if one is avai
 
 Although running jobs locally with `circleci` is very helpful, there are some limitations.
 
+### Setting Environment Variables
+
+We don't yet have an ideal solution for setting environment variables, but there is a workaround. Define which variables you want to export in a file within your repository (don't commit it - just have it in the directory). Then set BASH_ENV to that filename in your `.cirecleci.config.yml` file. The file will be automatically sourced on each command as long as BASH_ENV is correct.
```

Comment:
Suggest to be super clear this is for local builds:

"We don't yet have an ideal solution for setting environment variables on local builds, ..."

I've not had a chance to check, but if `BASH_ENV` is set in `.circleci/config.yml` what happens when that file doesn't exist on a non-local build?